simcon_project/simcon_project/settings.py

```diff
@@ -130,9 +125,18 @@
 
 USE_TZ = True
 
+# Email
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_USE_TLS = True
+EMAIL_PORT = 587
+EMAIL_HOST_USER = 'simcon.dev@gmail.com'
+EMAIL_HOST_PASSWORD = 'zhtsjuriiabkdahg'
```

Comment:
aaaaaAAAAAAAHHH NOOOOOOOOOOOOOO

Reply:
although first email goes to spam, the rest go to inbox

Reply:
What's up?